RestApi/Sepes.RestApi/Services/StudyService.cs

```diff
@@ -64,6 +66,42 @@ public async Task<Study> Save(Study newStudy, Study based)
             return study;
         }
 
+        public async Task<Study> DeletePod(Pod pod)
+        {
+            Study study = null;
+
+            foreach ( var iStudy in _studies.ToList())
+            {
+                if (iStudy.studyId == pod.studyId)
+                {
+                    study = iStudy;
+                }
+
+            }
+            
+            if (study.pods.Contains(pod))
+            {
+                await _podService.Delete(pod);
+                //Remove pod from study
+                var tempPods = study.pods.Remove(pod);
+                var studynew = study.ReplacePods(tempPods);
+                //Delete from database
+                await _db.UpdateStudy(studynew);
+                //Replace study in memory
+                _studies.Remove(study);
+                _studies.Add(studynew);
+                return studynew;
```

Comment:
There is no need to remove and replace pods to create a new study to save to memory. You can use the one sent from StudyController.

Reply:
Current implementation of delete has only a singular pod be passed from StudyController. So study in memory has to be modified to update memory. 
Besides StudyController passes on what is recieved from the frontend. For data safety I would assume it is bad practice to use frontends idea of the current state as the Word-of-God and such a change would need to be undone when sepes is made more compatible with multi user editing.